### PR TITLE
Floyd Warshall Algorithm In C (#2205)

### PR DESCRIPTION
**Pull request by p-soham11**
Fixes #2205 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [ ] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
Adding Floyd Warshall Algorithm in C

#### Languages Used:
C

#### Files Added:
- Floyd_Warshall_Algorithm.c

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by p-soham11** [comment]
Okay Mam
I will do it :)

**Comment by p-soham11** [comment]
Mam, I have made the changes accordingly.
Please guide me if anything else is needed to be changed. :)

File: Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.c [hunk 1]
```diff
@@ -0,0 +1,109 @@
+//C Program for Floyd Warshall Algorithm
+
+#include<stdio.h>
+
+//Number of vertices in the graph
+#define A 4
```
Comment (raghav-dalmia) [inline Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.c]: don't use macros to define a variable. use int data type.

File: Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.c [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+//Define INF(infinite) as a large value. This will be used for
+//the vertices that are not connected with each other
+#define INF INT_MAX
+
+//A function to print the solution matrix
+void printSolution(int dist[][A]);
```
Comment (raghav-dalmia) [inline Floyd_Warshall_Algorithm/Floyd_Warshall_Algorithm.c]: variable name should be in lower case or in camel case.

**Comment by raghav-dalmia** [comment]
Also, squash your commits.

**Pull request closed without merging**